hamocc/beleg_vars.F90

```diff
@@ -51,6 +51,9 @@ SUBROUTINE BELEG_VARS(kpaufr,kpie,kpje,kpke,kbnd,pddpo,prho,omask,      &
 
       ! local variables
       INTEGER :: i,j,k,l
+#ifdef cisonew
+      REAL :: rco213,rco214,beta13,beta14
```

Comment:
This bug was introduced when splitting beleg.F90 into beleg_vars.F90 and beleg_parm.F90. The variables rco213,rco214 are only used in the latter, but are still defined in the former. So they could be removed from beleg_parm.F90.